src/Liquid/AbstractBlock.php

```diff
@@ -164,7 +164,7 @@ protected function renderAll(array $list, Context $context)
 			}
 
 			if (is_array($value)) {
-				$value = htmlspecialchars(print_r($value, true));
+				$value = implode($value);
```

Comment:
This used to escape HTML; we have to keep it this way

Reply:
done